pdc_client/test_helpers.py

```diff
@@ -44,7 +44,7 @@ def __getitem__(self, key):
     class MockAPI(object):
         """Representation of mocked API.
 
-        This class replaces the PDCClient in code. Without any additional
+        This class replaces the PDCClientWithPage in code. Without any additional
```

Comment:
I would rather see separate small test for `PDCClientWithPage` (this class should be private anyway).

Reply:
All the test cases are using PDCClientWithPage now, because the pdc use a runner.py, and runner.py calls PDCClientWithPage.

Reply:
The docstring says that `MockAPI` is "Representation of mocked API" which would indicate `PDCClient`. The class `PDCClientWithPage` is just helper for `pdc` command.

Reply:
Yes, correct, I have changed that, please help to review.